### PR TITLE
Read Interop EXIF subdirectory

### PR DESCRIPTION
**Pull request by radarhere**
Helps #3270

https://www.awaresystems.be/imaging/tiff/tifftags/interoperabilityifd.html

### CODE REVIEW

File: Tests/test_file_jpeg.py [hunk 1]
```diff
@@ -255,6 +255,21 @@ def test_exif_gps_typeerror(self):
...
+
+        # Act
+        exif = im._getexif()
+
+        # Assert
+        self.assertEqual(exif["interop"], {
+            1: 'R98',
+            2: b'0100',
+            4097: 2272,
+            4098: 1704
```
Comment (hugovk) [inline Tests/test_file_jpeg.py]: Tip: including a comma means when the next one is added, the diff is smaller:
```suggestion
            4098: 1704,
```

File: src/PIL/JpegImagePlugin.py [hunk 2]
```diff
@@ -489,6 +489,18 @@ def _getexif(self):
         info = TiffImagePlugin.ImageFileDirectory_v1(head)
         info.load(file)
         exif[0x8825] = _fixup_dict(info)
+    # get interop
+    try:
+        # exif field 0xa005 is an offset pointer to the location
```
Comment (hugovk) [inline src/PIL/JpegImagePlugin.py]: Minor nits:
```suggestion
        # Exif field 0xa005 is an offset pointer to the location
```

File: src/PIL/JpegImagePlugin.py [hunk 3]
```diff
@@ -489,6 +489,18 @@ def _getexif(self):
...
+        # of the nested embedded interop exif ifd.
```
Comment (hugovk) [inline src/PIL/JpegImagePlugin.py]:
```suggestion
        # of the nested embedded interop Exif IFD.
```

**Comment by radarhere** [comment]
Incorporated into #3625

**Pull request closed without merging**